packages/webpack-config/src/helpers/paths.js

```diff
@@ -1,24 +1,23 @@
-'use strict';
```

Comment:
All the formatting changes are confusing and change our githistory. We need codeformatting rules asap.

Reply:
As discussed to make this clean and in a way pv-standards are defined I will add pv-eslint rules. Which will contain double quotes.